src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -51,6 +52,17 @@ public void WithScope(Action<Scope> scopeCallback)
         {
         }
 
+        /// <summary>
+        /// Returns a dummy transaction.
+        /// </summary>
+        public Transaction CreateTransaction(string name, string operation) =>
+            new Transaction(this, null);
```

Comment:
We should have a singleton NoOp instance here instead.
Having a disabled hub means we should reduce the instrumentations overhead